cmd/main.go

```diff
@@ -132,7 +132,11 @@ func main() {
 		NSXConfig: cf,
 	}
 
-	// Start the security policy controller.
+	if cf.CoeConfig.EnableVPCNetwork && commonService.NSXClient.NSXCheckVersion(nsx.VPC) {
+		log.V(1).Info("VPC mode enabled")
+		// Start controllers which only supports VPC
+	}
+	// Start the security policy controller, it supports VPC and non VPC mode
 	StartSecurityPolicyController(mgr, commonService)
 	staticroutecontroller.StartStaticRouteController(mgr, commonService)
```

Comment:
For other vpc related controllers: StartStaticRouteController, StartSubnetController,StartSubnetSetController, .StartSubnetPortController need to put in the condition: if cf.CoeConfig.EnableVPCNetwork

Reply:
ok